cross-platform/react-app/src/frontend/Screens/Hub/HubScreen.tsx

```diff
@@ -37,7 +33,7 @@ function getActiveProject() {
  * @see {@link getActiveProject}.
  * @param project The {@link Project} to set as the active project.
  */
-function saveActiveProject(project: Project) {
+function saveActiveProject(project: ITwin) {
```

Comment:
It's not good to store and iTwin, but then typecast it to a Project during load. (It might work, but it's still not good.)

Reply:
Note: I think you should just store the ITwin's id as a string in localStorage. Then the loading function can switch to checking if the JSON value is a string. If so, that's the ID, if not, it's old data, so it can return undefined.

Reply:
Fixed.